scripts/run_full.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+MVID=$1
+if [ "x${MVID}" = "x" ]; then
+    echo Must supply model version ID.
+    exit 4
+fi
+
+if [ $# -gt 1 ]; then
+    DEBUG=$2
+else
+    DEBUG=0
+fi
+
+if test -r ~/EPIAT_DIR
```

Comment:
This is new to me. It's a way to use your own environment. It uses the existence of a directory instead of using a configuration file in ~/.config/. Somehow I think the configuration file would be better? But we should discuss.

Reply:
I think you're the author of line 15?  

the next line is: EPI_DIR="$(cat ~/EPIAT_DIR)"

so, it appears that EPIAT_DIR would be a readable file that would contain the value for EPI_DIR?

Reply:
I suggest we could make a dotfile that redirects user code to use a particular install. This works meanwhile.